style/annotator-content-analysis.scss

```diff
@@ -0,0 +1,23 @@
+.survey-unit {
+  position: relative;
+  // top: 0;
+  right: 0;
+
+  &__topic {
+    font-size: 1.2em;
+    &--name {
```

Comment:
should this be double underscore instead of double hyphen? never seen this notation in DOM identifiers before...